#! /bin/sh /usr/share/dpatch/dpatch-run
## 931_libruby_suffix.dpatch by akira yamada <akira@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: changes name of ruby runtime library to libRUBYNAME-1.9.1.so* from libRUBYNAME.so*

@DPATCH@
Index: b/configure.in
===================================================================
--- a/configure.in	2010-07-20 21:46:15.000000000 +0900
+++ b/configure.in	2010-07-20 21:46:15.000000000 +0900
@@ -1702,7 +1702,7 @@
     ;;
 esac
 
-RUBY_SO_NAME='$(RUBY_INSTALL_NAME)'
+RUBY_SO_NAME='ruby-1.9.1'
 LIBRUBY_LDSHARED=$LDSHARED
 LIBRUBY_DLDFLAGS=$DLDFLAGS
 LIBRUBY_SO='lib$(RUBY_SO_NAME).so.$(MAJOR).$(MINOR).$(TEENY)'
@@ -2019,13 +2019,13 @@
 AC_ARG_WITH(sitedir,
 	    [  --with-sitedir=DIR      site libraries in DIR [[LIBDIR/ruby/site_ruby]]],
             [sitedir=$withval],
-            [sitedir='${libdir}/${RUBY_INSTALL_NAME}/site_ruby'])
+            [sitedir='${libdir}/ruby/site_ruby'])
 SITE_DIR=`eval echo \\"${sitedir}\\"`
 
 AC_ARG_WITH(vendordir,
 	    [  --with-vendordir=DIR    vendor libraries in DIR [[LIBDIR/ruby/vendor_ruby]]],
             [vendordir=$withval],
-            [vendordir='${libdir}/${RUBY_INSTALL_NAME}/vendor_ruby'])
+            [vendordir='${libdir}/ruby/vendor_ruby'])
 VENDOR_DIR=`eval echo \\"${vendordir}\\"`
 
 case "$target_os" in
@@ -2094,7 +2094,7 @@
 AC_ARG_WITH(rubyhdrdir,
 	    AS_HELP_STRING([--with-rubyhdrdir=DIR], [core headers in DIR [[INCLUDEDIR/RUBY_BASE_NAME-RUBY_VERSION]]]),
 	    [rubyhdrdir=$withval],
-	    [rubyhdrdir='${includedir}/${RUBY_INSTALL_NAME}-${ruby_version}'])
+	    [rubyhdrdir='${includedir}/ruby-${ruby_version}'])
 
 AC_ARG_WITH(sitehdrdir,
 	    [  --with-site-hdrdir=DIR  core headers in DIR [INCLUDEDIR]],
